tools/setup/setup_gcsfuse.sh

```diff
@@ -42,12 +42,18 @@ if [[ -d ${MOUNT_PATH} ]]; then
 fi
 
 mkdir -p $MOUNT_PATH
+MAXTEXT_VERSION=$(pip list | grep '^maxtext ' | awk '{print $2}')
+if [[ -z "$MAXTEXT_VERSION" ]]; then
+  MAXTEXT_VERSION="unknown"
+fi
+APP_NAME="maxtext-gcsfuse/$MAXTEXT_VERSION"
```

Comment:
Could we add both maxtext and grain versions here? Since the main use case for gcsfuse is for grain